src/main/java/duke/Task.java

```diff
@@ -0,0 +1,68 @@
+package duke;
+
+/**
+ * A class representing a task.
+ */
+public class Task {
+    protected String description;
+    protected boolean isDone;
+
+    /**a
```

Comment:
Small typo here.

Reply:
Thanks! Would not have caught that myself.